### ISSUE TITLE
Gifcurry snap won't launch on Ubuntu 20.04 Wayland

### ISSUE THREAD

**Issue by mjdubbs** [lettier/gifcurry#57]
Hello,
I get the following error message when I launch gifcurry gui 6.0.0 snap from the command line on ubuntu 20.04 wayland and it doesn't launch. It works fine on X

```
(gifcurry_gui:39816): Gdk-WARNING **: Wayland compositor does not support xdg_shell interface, not using Wayland display
(gifcurry_gui:39816): Gtk-WARNING **: cannot open display: :0
```

I think adding `DISABLE_WAYLAND: 1` to the snapcraft.yaml environment to fallback to XWayland could fix it. I tried testing it but I'm having a lot of trouble getting the snap to build.
Thanks

**Comment by lettier** [comment]
Hello,

Thank you for the suggestion.

I will look into it.

:+1:

**Comment by hyphone** [comment]
`DISABLE_WAYLAND: 1` to the snapcraft helps apps that don't support Wayland. Worked also for older Electron versions.

**Comment by ryanerwin** [comment]
I'm also trying to run Gifcurry on Ubuntu 20.04 under Wayland, but getting a coredump, possibly earlier than the Wayland error...

```
 ❯ ./gifcurry-6.0.0.0-x86_64_94815f8611347aaf6425ea3c83366c66.AppImage 
qt5ct: using qt5ct plugin

(gifcurry_gui:1970360): GLib-GIO-ERROR **: 13:11:49.066: Settings schema 'org.gtk.Settings.FileChooser' does not contain a key named 'show-type-column'
zsh: trace trap (core dumped)  ./gifcurry-6.0.0.0-x86_64_94815f8611347aaf6425ea3c83366c66.AppImage
```
I saw this question and answer about it, but on Ubuntu the default `/usr/share/glib-2.0/schemas/org.gtk.Settings.FileChooser.gschema.xml` does contain the `show-type-coumn` key:

https://askubuntu.com/questions/820733/firefox-hangs-because-sort-directories-first-setting-missing

**Comment by lettier** [comment]
Hello,

The `org.gtk.Settings.FileChooser.gschema.xml` issue has been resolved with the provided AppImage.

I can confirm Gifcurry itself is compatible with Ubuntu 20.04 on Wayland. 

![wayland](https://user-images.githubusercontent.com/1661343/87501792-fbe2e380-c62d-11ea-9f12-3ac4fb819a59.png)

A snap update is still being worked on.

:+1:

**Comment by mjdubbs** [comment]
I just tested the AppImage on Ubuntu 20.04 Wayland too. Works great!
Looking forward to the snap update
Thanks!
:+1:

**Comment by lettier** [comment]
Fixed in [6.0.1.0](https://github.com/lettier/gifcurry/releases/tag/6.0.1.0).

:+1:

**Comment by lettier** [comment]
Snap updated for Wayland.

![](https://user-images.githubusercontent.com/1661343/95816345-de84ea80-0cec-11eb-88ff-788a4e31bb9e.png)

:+1:
